tests/jenkins/TestPublishMinSnapshots.groovy

```diff
@@ -99,6 +100,7 @@ class TestPublishMinSnapshots extends BuildPipelineTest {
             return new Yaml().load(('tests/jenkins/data/opensearch-min-3.0.0-snapshot-darwin-build-manifest.yml' as File).text)
         })
         runScript('jenkins/opensearch/publish-min-snapshots.jenkinsfile')
+        assertThat(getCommands('sh', '/usr/local/bin/update-alternatives'), hasItem(containsString('/usr/local/bin/update-alternatives --set java `/usr/local/bin/update-alternatives --list java')))
```

Comment:
Shouldn't it also test that the `grep openjdk-17`

Reply:
oh, I thought it would be even better to make sure the code base contains the grep part dynamically by using the `/usr/local/bin/update-alternatives --list java' but I can definitelly change it to predfined openjdk-17 , please let me know, thanks.

Reply:
Yeah we are testing what you just said. We pass a specific test manifest here https://github.com/opensearch-project/opensearch-build/pull/4333/files#diff-46640feebd51b092f453f71a373c09d6181e3f990949941215e10323da16bac5R41 and https://github.com/opensearch-project/opensearch-build/pull/4333/files#diff-46640feebd51b092f453f71a373c09d6181e3f990949941215e10323da16bac5R49
so asserting would ensure it understands what we want.

Reply:
cool, makes sense, I have just got it updated accordingly, sorry for having you to do another review.. thanks